luzer/tests/test_unit.lua

```diff
@@ -174,6 +174,12 @@ assert(p1 >= 0 and p2 >= 0)
 assert(p1 <= 1 and p2 <= 1)
 assert(p1 ~= p2)
 
+-- luzer.FuzzedDataProvider.oneof()
+fdp = luzer.FuzzedDataProvider("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+assert(type(fdp.oneof) == "function")
+local n = fdp:oneof({1, 2})
```

Comment:
Minor: I would rather use `3, 4` instead since it is easier to debug the LuaC part: `1, 2` are matched with their indexes. Hence, it may be confusing what you have on the Lua stack: the index or the value from the table. Feel free to ignore, though.

Reply:
Thanks, I like suggestion!

The diff also defines boundaries in variables.

```diff
--- a/luzer/tests/test_unit.lua
+++ b/luzer/tests/test_unit.lua
@@ -177,8 +177,10 @@ assert(p1 ~= p2)
 -- luzer.FuzzedDataProvider.oneof()
 fdp = luzer.FuzzedDataProvider("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
 assert(type(fdp.oneof) == "function")
-local n = fdp:oneof({1, 2})
-assert(type(n) == "number" and (n == 1 or n == 2))
+local a = 3
+local b = 4
+local n = fdp:oneof({a, b})
+assert(type(n) == "number" and (n == a or n == b))
 
 local function custom_mutator(data, size, max_size, seed)
     assert(type(data) == "string")
```